### ISSUE TITLE
Error: Method not found: 'UnmodifiableUint8ListView'.

### ISSUE THREAD

**Issue by user97116** [brendan-duncan/archive#350]
/home/amar/.pub-cache/hosted/pub.dev/archive-3.4.9/lib/src/bzip2/bzip2.dart:5:7: Error: Method not found: 'UnmodifiableUint8ListView'.
      UnmodifiableUint8ListView(Uint8List(0));
      ^^^^^^^^^^^^^^^^^^^^^^^^^
/home/amar/.pub-cache/hosted/pub.dev/archive-3.4.9/lib/src/bzip2/bzip2.dart:7:7: Error: Method not found: 'UnmodifiableUint32ListView'.
      UnmodifiableUint32ListView(Uint32List(0));
      ^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/amar/.pub-cache/hosted/pub.dev/archive-3.4.9/lib/src/bzip2/bzip2.dart:9:7: Error: Method not found: 'UnmodifiableInt32ListView'.
      UnmodifiableInt32ListView(Int32List(0));
      ^^^^^^^^^^^^^^^^^^^^^^^^^
Target kernel_snapshot_program failed: Exception

**Comment by user97116** [comment]
on dart 3.5.0

**Comment by proshin-roman** [comment]
@user97116 the issue seems to be still existing - why have you closed the ticket?

Here is an error message when building for iOS:
```
Error (Xcode): ../../../../.pub-cache/hosted/pub.dev/archive-3.4.9/lib/src/bzip2/bzip2.dart:5:7: Error: Method not found: 'UnmodifiableUint8ListView'.
```

Flutter Doctor:
```
[✓] Flutter (Channel stable, 3.24.0, on macOS 14.5 23F79 darwin-x64, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
...
```

Dart version:
```
Dart SDK version: 3.5.0 (stable) (Tue Jul 30 02:17:59 2024 -0700) on "macos_x64"
```

**Comment by proshin-roman** [comment]
Ok, it's a good idea to first check other closed tickets - found #346 and then also noticed that I'm using version `3.4.9` of the package, while the latest is `3.6.1`, and that's because I have another package that is locked on 3.4.9: https://github.com/justkawal/excel/issues/364
